### PR TITLE
[NEW] sale_procurement_analytic: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
This module depends of #62 and replaces #61

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by carlosdauden** [comment]
Rebased

**Comment by pedrobaeza** [comment]
cc @Tecnativa

**Comment by rafaelbn** [comment]
Please @carlosdauden rebase there is not travis

Thanks

File: sale_procurement_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Sale Procurement Analytic
+=========================
+
+This module sets analytic account procurement from sale order.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
```
Comment (rafaelbn) [inline sale_procurement_analytic/README.rst:15]: Go to setting -> Local Modules and install module accounting

File: sale_procurement_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#. Enable *Analytic Accounting for Sales*.
+
+Usage
+=====
+
+#. Go to *Sales -> Sales Orders* and create new SO witch contains product/s
+   with *Suppliers* info and *Make To Order* *Route* are set.
+#. Set *Contract / Analytic* field.
+#. *Confirm Sale*
+
```
Comment (rafaelbn) [inline sale_procurement_analytic/README.rst]: What else after confirming sale?

File: sale_procurement_analytic/README.rst (hunk 1 above)
Comment (pedrobaeza) [inline sale_procurement_analytic/README.rst:15]: Isn't auto-installed as dependency?

File: sale_procurement_analytic/README.rst (hunk 2 above)
Comment (rafaelbn) [inline sale_procurement_analytic/README.rst]: Go to Warehouse -> Schedulers -> Procurements and check _Analytic Account_ in _Extra Information_ tab

**Comment by rafaelbn** [comment]
@carlosdauden there is not field "Analytic Account" in procurement.order, please review it. Thanks!

**Comment by rafaelbn** [comment]
@carlosdauden there is not field "Analytic Account" in procurement.order, please review it. Thanks!

**Comment by carlosdauden** [comment]
@rafaelbn Fixed in https://github.com/OCA/account-analytic/pull/66

**Comment by rafaelbn** [comment]
I tested https://github.com/OCA/account-analytic/pull/66 this PR is :+1:  Thanks!

**Comment by rafaelbn** [comment]
This PR is ready to be merged. Please @OCA/accounting-maintainers could you take a look into here? Thanks

**Comment by pedrobaeza** [comment]
No, it's not ready. The permission for seeing the field should be added here, not in https://github.com/OCA/account-analytic/pull/66.

Maybe you need to inject this dependency via fields_view_get method to avoid to lose the changes if the other module (purchase_...). Look an example here: https://github.com/OCA/bank-payment/blob/35c508476b5cfbc3c3b13f366f1cfe9b0d7bc34a/account_banking_payment_export/models/account_invoice.py#L20-L48

File: sale_procurement_analytic/README.rst (hunk 1 above)
Comment (yajo) [reply sale_procurement_analytic/README.rst:15]: Actually not. Either add it as @rafaelbn says, or add the dependency please.

File: sale_procurement_analytic/README.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+To configure this module, you need to:
+
+#. Go to *Settings > Configuration > Accounting > Analytic Accounting*.
+#. Enable *Analytic Accounting for Sales*.
+
+Usage
+=====
+
+#. Go to *Sales -> Sales Orders* and create new SO witch contains product/s
```
Comment (yajo) [inline sale_procurement_analytic/README.rst]: s/witch/which :laughing:

File: sale_procurement_analytic/README.rst (hunk 2 above)
Comment (yajo) [reply sale_procurement_analytic/README.rst]: Done all this, I still see no visible change, not even where @rafaelbn says.

I see no views here... Shouldn't you add the field to a view? BTW instructions do not say if you need to confirm sale or what...

Comment (yajo) [reply sale_procurement_analytic/README.rst]: OK I found it. I have to enable too _Settings > Configuration > Accounting > Analytic Accounting > Purchase >  Analytic accounting for purchases_ and then do https://github.com/OCA/account-analytic/pull/64/files#r71774103.

Not sure if all that should be autoinstalled or added to docs, but either is required for sure.

**Comment by rafaelbn** [comment]
Rebuilding for testing, @carlosdauden some comments pending

**Comment by rafaelbn** [comment]
Hi @carlosdauden @pedrobaeza what's going on with this PR? https://github.com/OCA/account-analytic/pull/66 is closed without merge so this PR fails. Please review in order to test it, thanks!

**Comment by pedrobaeza** [comment]
Discarding in favor of a new 10.0 version.

**Pull request closed without merging**